db/languages.yml

```
@@ -138,3 +138,21 @@ j:
     j8:
       :name: "J 8.0.3"
       :interpreter: /usr/bin/ijconsole-8.0.3
+
+csharp:
+  :name: "C#"
+  :lexer: csharp
+  :compiled: yes
+  :interpreted: yes
+  :current: csharp8
+  :extension: .cs
+  :source_filename: program
```

```suggestion
  :source_filename: program
  :exe_extension: .exe
```
Needs this otherwise it gives a runtime error – `dotnet exec needs a managed .dll or .exe extension`.